src/components/text-input-with-copy-button/TextInputWithCopyButton.js

```diff
@@ -150,6 +150,9 @@ class TextInputWithCopyButton extends React.PureComponent<Props, State> {
     handleFocus = (event: SyntheticEvent<>) => {
         if (this.copyInputRef) {
```

Comment:
I realize this code was already duplicated, but we're adding to it a bit. Could this method also call `this.performAutoFocus()` directly?

Reply:
It certainly could.